react/Avatar/index.jsx

```diff
@@ -46,9 +56,10 @@ export const Avatar = ({ text, textId, image, size, className, style }) => {
       className={cx(
         styles['c-avatar'],
         size !== 'medium' ? styles[`c-avatar--${size}`] : '',
+        disabled ? styles['c-avatar--disabled'] : '',
         className
       )}
-      style={text ? colored : style}
+      style={text ? colored : image ? { backgroundColor: 'white' } : style}
```

Comment:
why white? why not `transparent` ?

Reply:
Parce qu'on a une opacité inférieur à 100% sur les images. Avec un fond transparent, on n'obtient pas le gris clair attendu. De plus, si les avatars se chevauchent, ce qui est le cas dans Drive, on voit l'autre avatar en-dessous.

Reply:
Je trouve ça compliqué à lire que le white est parce qu'on utilise `grayscale` et une opacity sur l'image, mais aussi qu'on définisse un background --silver et que juste après on le surcharge en white. 

Je me demande s'il ne faudrait pas créer des classes css spécifiques pour chaque cas ? 

Parce que d'ailleurs, si on est en text mais disabled on va avoir : 
un style inline avec "backgroundColor: --silver" et "c-avatar--disabled qui lui même ajoute le background silver)" non ?

Reply:
> Parce que d'ailleurs, si on est en text mais disabled on va avoir :
un style inline avec "backgroundColor: --silver" et "c-avatar--disabled qui lui même ajoute le background silver)" non ?

Oui tout à fait.
Je regarde pour refacto un peu, c'est effectivement un composant pas très lisible

Reply:
J'ai simplifié un peu, mais pour le fond coloré du text j'ai pas trouvé mieux. Soit il faut utiliser une classe, mais dupliquer le tableau des couleurs dans le stylus, soit il faut utiliser les classe utilitaires directement mais comme elles sont en important ça override le style des props (alors qu'on veut le comportement inverse).